packages/open-next/src/core/routing/i18n/index.ts

```diff
@@ -31,7 +31,7 @@ export function detectDomainLocale({
   detectedLocale?: string;
 }): DomainLocale | undefined {
   const i18n = NextConfig.i18n;
-  if (!i18n || i18n.localeDetection === false || !i18n.domains) {
+  if (!i18n || !i18n.domains) {
```

Comment:
Which I think is the same as:
```suggestion
  if (!i18n) {
```

Reply:
Or `const domain = NextConfig.i18n?.domain; if {!domain)`